pangeo_forge_runner/bakery/flink.py

```diff
@@ -150,13 +150,99 @@ class FlinkOperatorBakery(Bakery):
         """,
     )
 
+    enable_job_archiving = Bool(
+        False,
+        config=True,
+        help="""
+            Enable the ability for past jobs to be archived so the job 
+            manager's REST API can still return information after completing
+            """,
+    )
+
+    job_archive_efs_mount = Unicode(
+        "/opt/history/jobs",
+        config=False,
+        help="""
+        A non-configurable mount path showing where past jobs are
+        archived so the job manager's REST API can still return
+        information after completing
+        
+        The reason this is non-configurable is b/c the same mount path
```

Comment:
We should definitely keep this configurable, as pangeo-forge-runner should be able to run with all sorts of setups, and *not* be deeply tied into just this one particular terraform setup that we have. The documentation here should instead say what exactly should be configured in the *operator* for this to work, and what paths should match. 

In general, keeping these loosely coupled helps increase adoption of open source projects, as often end users will have infrastructure they can only co-opt but not fully re-use! For example, another organization with its own terraform could steal our terraform and modify, but it will have to *fit into* theirs in some way or form. There's currently no strong tie between the runner and the cloud federation terraform code, so I'd like us to not add more.